lldb/source/Commands/CommandObjectBreakpoint.cpp

```diff
@@ -319,6 +316,12 @@ class CommandObjectBreakpointSet : public CommandObjectParsed {
           error_context = "Unknown language type for exception breakpoint";
           break;
         default:
+          if (Language *languagePlugin = Language::FindPlugin(language)) {
```

Comment:
You could drop the braces from the first if I think, but this is already in so many sets of braces I'd keep them for clarity's sake.

Reply:
Within switch statements I tend to be very paranoid with braces